llvm/test/Transforms/LoopVectorize/X86/cse-bitcast.ll

```diff
@@ -0,0 +1,77 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --filter-out-after "^scalar.ph" --version 5
+; RUN: opt %s -passes=loop-vectorize -S | FileCheck %s
```

Comment:
I think the issue here is specifically for interleave groups.

You should be able to make this test target-independent by enabling interleave groups with a flag (see the interleaved-accesses.ll file).

Reply:
You're right! Using `-force-vector-width=4 -force-vector-interleave=2 -enable-interleaved-mem-accesses=true` makes the test target-dependent.